k8s/generate_code.sh

```diff
@@ -37,14 +37,14 @@ if [[ ! -d "${GOPATH}/src/k8s.io/code-generator" ]]; then
 fi
 
 # https://github.com/kubernetes-sigs/controller-tools/tree/v0.7.0/cmd/controller-gen
-if ! command -v ${GOPATH}/bin/controller-gen > /dev/null; then
```

Comment:
this is to revert the change from my previous PR
this change is only for my own env